massa-graph/src/export_active_block.rs

```diff
@@ -14,7 +15,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ExportActiveBlock {
     /// The id of the block.
-    pub block: Block,
+    pub block: BlockId,
```

Comment:
When exporting the structure, it might make sense to actually include the block, since the user of the exported structure might want that info as well(and shared storage wouldn't necessarily be available to retrieve the block at that point). 

cc @AureliaDolo @damip @yvan-sraka

Reply:
It's used during bootstrapping. So indeed, here the full block seems needed.

Reply:
ok

Reply:
Changes made